examples/text_me.py

```diff
@@ -0,0 +1,59 @@
+# /// script
+# dependencies = ["fastmcp"]
```

Comment:
This *should* be unecessary as fastmcp always adds itself as a dependency but 👍 good practice. I did learn that if you have a script declaration like this it overrules all other dependencies though